app/Http/Controllers/Moderation/TicketsController.php

```diff
@@ -145,7 +175,13 @@ public function blacklist()
 
     public function blacklistAdd(Request $request)
     {
+        try {
         $user = User::where('id', $request->user_id)->first();
```

Comment:
When using first(), it wont throw an exception when there is no user. It will just leave the variable empty. Should be changed to firstOrFail as its mandatory i guess?